extensions/wikia/PageHeader/styles/index.scss

```diff
@@ -33,4 +40,34 @@
 		font-weight: bold;
 		text-align: right;
 	}
+
+	&__categories {
+		@include align-items(flex-start);
+		@include flexbox();
+		font-size: 12px;
+		padding-top: 12px;
+
+		&__in {
+			font-weight: 600;
+			padding-right: 4px;
+		}
+
+		&-links {
+			color: $color-links;
+			font-style: italic;
+			font-size: 12px;
+			line-height: 16px;
+			word-break: break-word;
```

Comment:
We'll use` word-break: break-all; ` as you sugested, becaues it works on every browser.